src/main/java/org/umssdiplo/automationv01/core/managepage/listItemType/ListItemType.java

```diff
@@ -0,0 +1,4 @@
+package org.umssdiplo.automationv01.core.managepage.listItemType;
+
+public class ListItemType {
```

Comment:
you need to use extends
after that it should have in the constructure some webelment that we need to wait.